protostar/commands/test/test_collector.py

```diff
@@ -47,8 +67,13 @@ def log(self, logger: Logger):
             else:
                 logger.warning("No cases found")
 
-    def __init__(self, starknet_compiler: StarknetCompiler) -> None:
+    def __init__(
+        self,
+        starknet_compiler: StarknetCompiler,
+        default_test_suite_glob: Optional[str] = None,
```

Comment:
I can move this parameter it to the `collect` function. This must be done in this class because the None value would propagate to the TestSuite::path.